File: C++17/백준/Bronze/25372. 성택이의 은밀한 비밀번호/성택이의 은밀한 비밀번호.cc
#include <iostream>
#include <string>
using namespace std;

int main (void) {
    int c;
    if (!(cin >> c)) return 0;

    for (int i = 0; i < c; i++) {
        string pwd;
        cin >> pwd;

        if (pwd.length() >= 6 && pwd.length() <= 9) {
            cout << "yes" << endl;
        } else {
            cout << "no" << endl;
        }
    }

    return 0;
}

File: Java/백준/Bronze/25372. 성택이의 은밀한 비밀번호/성택이의 은밀한 비밀번호.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int c = sc.nextInt();
        
        for (int i = 0; i < c; i++) {
            String pwd = sc.next();
            
            if (pwd.length() >= 6 && pwd.length() <= 9) {
                System.out.println("yes");
            } else {
                System.out.println("no");
            }
            
        }
        
        sc.close();
    }
}